spec/lib/terraforming/resource/eip_spec.rb

```diff
@@ -0,0 +1,115 @@
+require "spec_helper"
+
+module Terraforming
+  module Resource
+    describe EIP do
+      let(:client) do
+        Aws::EC2::Client.new(stub_responses: true)
+      end
+
+      let(:eips) do
+        [
+          {
+            domain: "vpc",
+            instance_id: "i-12345678",
+            network_interface_id: "eni-12345678",
+            association_id: "eipassoc-98765432",
+            network_interface_owner_id: "123456789012",
+            public_ip: "12.34.56.78",
+            allocation_id: "eipalloc-87654321",
+            private_ip_address: "1.1.1.1",
+          },
+          {
+            domain: "vpc",
+            network_interface_id: "eni-23456789",
+            association_id: "eipassoc-87654321",
+            network_interface_owner_id: "234567890123",
+            public_ip: "2.2.2.2",
+            allocation_id: "eipalloc-76543210",
+            private_ip_address: "9.9.9.9",
+          },
+          {
+            public_ip: "3.3.3.3",
+            domain: "vpc",
+            allocation_id: "eipalloc-33333333",
+          },
+        ]
+      end
+
+      before do
+        client.stub_responses(:describe_addresses, addresses: eips)
+      end
+
+      describe ".tf" do
+        it "should generate tf" do
+          expect(described_class.tf(client: client)).to eq <<-EOS
+resource "aws_eip" "eipalloc-87654321" {
+    instance             = "i-12345678"
+    vpc                  = true
+}
+
+resource "aws_eip" "eipalloc-76543210" {
+    network_interface_id = "eni-23456789"
+    vpc                  = true
+}
+
+resource "aws_eip" "eipalloc-33333333" {
+}
```

Comment:
This tf expects a resource in VPC, so this should be contain `vpc = true` .

```
resource "aws_eip" "eipalloc-33333333" {
    vpc                  = true
}
```

(Actually it causes a diff at `terraform plan` when I tested to export non-associated Elastic IP with my account...)